sqlglot/parser.py

```diff
@@ -1239,6 +1246,45 @@ def _parse_table_sample(self):
             seed=seed,
         )
 
+    def _parse_pivot(self):
+        if self._match(TokenType.PIVOT):
+            unpivot = False
+        elif self._match(TokenType.UNPIVOT):
+            unpivot = True
+        else:
+            return None
+
+        summary = None
+        value = None
+        expressions = None
+
+        self._match_l_paren()
+        if unpivot:
+            summary = self._parse_column()
+        else:
+            summary = self._parse_function()
+
+        if not self._match(TokenType.FOR):
+            self.raise_error("Expecting FOR")
+
+        value = self._parse_column()
```

Comment:
why not just call parse IN?

Reply:
I tried that and it didn't work for some reason. I can check again.

Reply:
parse IN will work. I will refactor.